server/src/infra/migrations/1703950964498-DefaultOnboardingForExistingInstallations.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DefaultOnboardingForExistingInstallations1703950964498 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const adminCount = await queryRunner.query(`SELECT COUNT(*) FROM users WHERE "isAdmin" = true`);
+    if (adminCount[0].count > 0) {
+      await queryRunner.query(`UPDATE users SET "showOnboarding" = false WHERE "isAdmin" = true`);
+    }
```

Comment:
Do we need the conditional here? We can just always run the UPDATE and it'll no-op if there are no admins, right?